ociinstaller/versionfile/versionfile.go

```diff
@@ -22,7 +23,7 @@ type VersionFile struct {
 }
 
 type InstalledVersion struct {
-	Name            string `json:"name"`
+	Name            string `json:"-"` // the fully qualified name of the plugin - auto populated
```

Comment:
should we just remove the `json` tag altogether?


I don't like comments at the end of lines - please put above

Reply:
Can't - since `Name` is a public field, the `encoder` will simple use `Name` for the field in the JSON as well.

Reply:
hmm I see - in which case this feels a bit hacky...

maybe you are right and an array is better? what are your thoughts?